pkg/gadgettracermanager/containers-map/bpf/containers-map.c

```diff
@@ -12,7 +12,6 @@ struct {
 	__type(key, __u64);
 	__type(value, struct container);
 	__uint(max_entries, MAX_CONTAINERS_PER_NODE);
-	__uint(pinning, LIBBPF_PIN_BY_NAME);
```

Comment:
By the way, this is fixing a bug: if gadget-tracer-manager crashes and is restarted, the map was not reinitialized before, so it could contains data about old containers. With this PR, we create a new map, so this bug is fixed.